guides/GITHUB.md

```diff
@@ -0,0 +1,334 @@
+# Getting Started with GitHub
+
+What is GitHub, and how do you use it to interact with and contribute to SourceCred?
+This guide will give you an introduction to GitHub and SourceCred.
+
+## Why do we use GitHub?
+
+For now, think of GitHub as a place to collaborate. 
+If you had a bunch of documents that you wanted to collaborate on, what would you do?
+You might consider putting them in a shared file service like [Google Docs](https://docs.google.com/), or perhaps keeping them as Word documents on your local machine. 
+Although these platforms offer general collaborative editing capabilities, GitHub
+offers a bit more:
+
+### Permissionless collaboration
+
+Anyone can propose changes. You don't need to be given permission to view or edit
+a document, as is required by some of the platforms mentioned previously.
+
+### More advanced reviewing tools
+
+When you open up a Google Doc, you might be able to comment or write in revision mode, but
+when you do so, you are working with the entire document. GitHub allows
+review on the level of change. Along with this more detailed ability to edit,
+there is an integrated approval workflow, straightforward visualizations of what changed 
+(these are called "diffs"), and powerful automatic testing (called continuous integration or CI).
+
+### More detailed history
+
+You are going to have basic questions like:
+
+ - Who added this piece?
+ - When and why was this added?
+
+And GitHub has built-in tools to do this (one such action is called [git blame](https://help.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file).)
+
+### Easy for many people to collaborate at once
+
+You can work in your own "alternate reality" without interfering with other work, even simultaneously. When ready these alternatives can be merged back together.
+You can also work on changes out of order, although this is more of an advanced topic.
```

Comment:
Why is there an order? I'm not sure what kind of order there would be for editing, other than top to bottom. A bit unclear to a non-tech person like me :)

Reply:
You're right - there isn't any order ingrained, I was thinking of how in some cases developers have a sense of when they want changes to go in, but that's probably overkill for this introduction.

Reply:
For an example, imagine you have a library with a bunch of PRs coming in for new features intended for the next version. You also might have PRs coming in for bug fixes for the current. Both of those sets of work can go on at the same time, or even the "for the next version" work could come before the bug fix work. This is what I meant by out of order, you don't absolutely have to do what chronologically needs to be merged into master first.

Reply:
(Note that I'm just removing this line).